### ISSUE TITLE
Run test instructions?

### ISSUE THREAD

**Issue by lenyabloko** [guyemerson/sem-func#1]
Are you planning to provide any instructions to run your code and reproduce the results.

**Comment by guyemerson** [comment]
I agree the code is not very user-friendly right now!  I can keep you posted about updates to the code and user instructions, but this is probably several months away.  If you have specific experiments you would like to run and a limited time frame, I would be happy to discuss (email might be best).

**Comment by lenyabloko** [comment]
Thank you. I will keep that in mind.
